ygot/struct_validation_map.go

```diff
@@ -906,6 +912,73 @@ func validateMap(srcField, dstField reflect.Value) (*mapType, error) {
 	return &mapType{key: st.Key(), value: st.Elem()}, nil
 }
 
+// copyOrderedMap copies srcField into dstField. Both srcField and dstField are
```

Comment:
nit: `srcField` isn't a parameter so this comment is a little hard to follow

Reply:
Done.